server/src/main/java/io/druid/server/router/PriorityTieredBrokerSelectorStrategy.java

```diff
@@ -50,18 +49,10 @@ public PriorityTieredBrokerSelectorStrategy(
 
     if (priority < minPriority) {
       return Optional.of(
-          Iterables.getLast(
-              tierConfig.getTierToBrokerMap().values(),
-              tierConfig.getDefaultBrokerServiceName()
-          )
+          tierConfig.getDefaultBrokerServiceName()
```

Comment:
Logging here might be helpful.

Reply:
`TieredBrokerSelectorStrategy` doesn't have a logger. Should I add a logger to this class?

Reply:
is the log for queries which don't fall in min-max priority range configured ? that log might end up being too noisy. in many clusters cluster operators and user teams are different and users might send such queries.